.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,19 @@
-<!--
-Thank you for your interest in and contributing to Kibana! There
-are a few simple things to check before submitting your pull request
-that can help with the review process. You should delete these items
-from your submission, but they are here to help bring them to your
-attention.
-
-- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
-- Have you followed the [contributor guidelines](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md)?
-- If submitting code, have you included unit tests that cover the changes?
-- If submitting code, have you tested and built your code locally prior to submission with `yarn test && yarn build`?
-- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
--->
\ No newline at end of file
+## Summary
+
+Summarize your PR. If it involves visual changes include a screenshot or gif.
+
+### Checklist
+
+Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.
+
+- [ ] This was checked for cross-browser compatability, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
```

Comment:
I think this is great in theory... but I do not test against IE11 because I'm on a mac. :(  Is this really the expectation that all developers have IE running on a VM that they can verify against?  I kinda doubt most people do this... interesting to hear thoughts.

Reply:
It's easy enough to get a VM from Microsoft with Windows/IE already set up to do this.  I think we should all be doing it.

Reply:
I know that most people don't do it which is why it needs to be on here :) . We either support IE11 or we don't. If we do, anyone who contributes new code should be self-reviewing their content for it.

It's the same reason accessibility. docs and i18n are on here: they generally are common things people like to skip or avoid.

Reply:
FYI, a while back I wrote up this doc about how to set up a VM + Putty so you can access the same Kibana instance you're running on your macOS host: https://docs.google.com/document/d/1Wvw0aDw9yheQBVJCZocrSDwFT2F4pEVdF56KhGDG90c/edit#

Reply:
That's awesome @walterra!  I wonder if we should put that info in our repo, in a readme, then we could link to it to help folks out who may not know how to test IE11 on a mac.